c_cxx/QNN_EP/README.md

```diff
@@ -1,5 +1,19 @@
+## Prerequisites
+- Qualcomm ARM Device - e.g. Surface Pro9 5G, Windows Dev Kit 2023
```

Comment:
QNN CPU backend doesn't require Qualcomm ARM device. it can run on x64. 
QC is providing us a landing page with docs and QNN download link at 
https://qpm.qualcomm.com/main/tools/details/qualcomm_ai_engine_direct

Reply:
Interesting. This leads to "Qualcomm® AI Engine Direct SDK". Is this a rebranding or how is this different than say the QNN SDK. QNN SDK (Qualcomm® Neural Network (NN) Software Development Kit (SDK)) from Qualcomm CreatePoint 
            - Example: Qualcomm_NN_SDK.WIN.2.0 Installer Preview

I can include the link if it's the right one and installs

Reply:
> Interesting. This leads to "Qualcomm® AI Engine Direct SDK". Is this a rebranding or how is this different than say the QNN SDK. QNN SDK (Qualcomm® Neural Network (NN) Software Development Kit (SDK)) from Qualcomm CreatePoint - Example: Qualcomm_NN_SDK.WIN.2.0 Installer Preview
> 
> I can include the link if it's the right one and installs

that's a good question re: naming of the SDK. there was no rebranding AFAIK, let me find out why it's named that way and get back to you.

Reply:
confirmed "Qualcomm AI Engine Direct SDK" is a marketing name. the package is still QNN. We won't change the name of the EP and code references. Our docs can refer to the marketing name followed by (QNN SDK) to reduce the confusion for users.

Reply:
Just changed to referencing new PR docs https://github.com/microsoft/onnxruntime/pull/16053